tajo-storage/tajo-storage-common/src/main/java/org/apache/tajo/storage/LazyTuple.java

```diff
@@ -134,7 +134,7 @@ else if (textBytes.length <= fieldId) {
       }
       textBytes[fieldId] = null;
     } else {
-      //non-projection
+      values[fieldId] = NullDatum.get(); //non-projection
```

Comment:
reverted this part. I've hardly noticed tajo expects null for not-picked and byte[0] for not-existing picked.